bot/utils/converters.py

```diff
@@ -14,3 +18,70 @@ async def convert(self, ctx: discord.ext.commands.Context, argument: str) -> dis
             argument = argument[1:-1]
 
         return await super().convert(ctx, argument)
+
```

Comment:
[Caught another Converter](https://github.com/python-discord/sir-lancebot/blob/05ba3de5c99b30a8eba393da1460fae255373457/bot/exts/evergreen/snakes/_converter.py#L16)

Reply:
I think that this can't be used anywhere else so I'll just leave it.